src/features/survey-forms/survey-forms-header.tsx

```diff
@@ -3,7 +3,7 @@ import { PageTitle } from "@components/shared/page-title"
 export const SurveyFormsHeader = () => {
   return (
     <div className='flex flex-col md:flex-row justify-between gap-4'>
-      <PageTitle>Survey Forms</PageTitle>
+      <PageTitle>Test Survey Sample</PageTitle>
```

Comment:
This header is for the survey forms (the list) page, please create another file for the view survey form header inside features/survey-forms/[id]

Reply:
done